### PR TITLE
Basic Structure

### PR DESCRIPTION
**Pull request by VitorGuedesMadeira**
### Blog app - creating a data model

✅  [Project Requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/creating_data_model_project_v1_1.md)

**_Learning objectives:_**
✔️ Install Ruby on Rails framework.
✔️ Use Ruby Gems as a software packages system.
✔️ Understand what ORM is.
✔️ Use database migration files to maintain database schema.
✔️ Install and check linters.
✔️ Add a descriptive README file.

**_author:_** [Vitor Guedes Madeira](https://github.com/VitorGuedesMadeira)
deadpool-clapping

### CODE REVIEW

File: db/migrate/20221018195032_create_post.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class CreatePost < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.integer :comments_counter
+      t.integer :likes_counter
+
+      t.timestamps
+    end
```
Comment (mikethreels) [inline db/migrate/20221018195032_create_post.rb:10]: - [ ] there should be a reference to Author in the post in the form of a foreign key.

File: db/migrate/20221018194832_create_comment.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class CreateComment < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text
+
+      t.timestamps
+    end
```
Comment (mikethreels) [inline db/migrate/20221018194832_create_comment.rb:7]: - [ ] there should be a reference to Author and post in the comment table in the form of a foreign key.

File: db/migrate/20221018195116_create_like.rb [hunk 3]
```diff
@@ -0,0 +1,8 @@
+class CreateLike < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+
+      t.timestamps
+    end
```
Comment (mikethreels) [inline db/migrate/20221018195116_create_like.rb:6]: - [ ] there should be a reference to Author and post in the like table in the form of a foreign key.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Vitor, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20221018195032_create_post.rb (hunk 1 above)
Comment (VitorGuedesMadeira) [reply db/migrate/20221018195032_create_post.rb:10]: @mikethreels Hello! Can you please check db/schema.rb? All of them already have an implemented FK. You can check it on db/migrate files where they adding the foreign keys, not the ones creating it

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @VitorGuedesMadeira, 

### Good job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### N.A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**